### PR TITLE
Refacto doc

### PR DESCRIPTION
**Pull request by sebastianMindee**
## Description

* :memo: add missing doc
* :coffin: remove outdated guide doc
* :recycle: rename some classes to properly reflect API specs

## Types of changes

- [x] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires a change to the official Guide documentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,6 +44,28 @@ result = mindee_client.parse(
 puts result.document
 ```
 
+**Note:** Files can also be loaded from:
+
+A URL (`https`):
+```js
```
Comment (ianardee) [inline README.md:50]: should be `rb` for all the examples

File: docs/getting_started.md [hunk 2]
```diff
@@ -1,8 +1,11 @@
...
+title: Ruby Getting Started
+---
+This guide will help you get the most out of the Mindee Ruby client library to easily extract data from your documents.
 
-The Ruby client supports [Invoice](https://developers.mindee.com/docs/ruby-invoice-ocr), [receipt](https://developers.mindee.com/docs/ruby-receipt-ocr),  [passport](https://developers.mindee.com/docs/ruby-passport-ocr),  OCR APIs and [custom-built API](https://developers.mindee.com/docs/ruby-api-builder) from the API Builder.
-
-You can view the source code on [GitHub](https://github.com/mindee/mindee-api-ruby).
+> 📘 **Info**
+> 
+> The library is typed-hinted using sorbet.
```
Comment (ianardee) [inline docs/getting_started.md:8]: I'm not seeing the sorbet dependencies in the gemfile, and don't see any signature specs in ruby files.
So doesn't look like this is actually true...

**Review by ianardee — CHANGES_REQUESTED** [review]

File: docs/getting_started.md (hunk 2 above)
Comment (sebastianMindee) [reply docs/getting_started.md:8]: Yeah I think I hallucinated that one hard :laughing:

File: README.md (hunk 1 above)
Comment (sebastianMindee) [reply README.md:50]: Woops :grimacing:

**Review by ianardee — APPROVED** [review]

**Pull request merged**